src/Misc/Part.cpp

```diff
@@ -179,20 +179,24 @@ static const Ports partPorts = {
         [](const char *, RtData &d)
         {
             Part *p = (Part*)d.obj;
+            int err;
             if (p->loaded_file[0] == '\0') {  // if part was never loaded or saved
                 time_t rawtime;     // make a new name from date and time
                 char filename[32];
                 time (&rawtime);
                 const struct tm* timeinfo = localtime (&rawtime);
                 strftime (filename,23,"%F_%R.xiz",timeinfo);
-                p->saveXML(filename);
-                fprintf(stderr, "Part %d saved to %s\n", (p->partno + 1), filename);
+                err = p->saveXML(filename);
+                fprintf(stderr, "Part %d saved to %s: %s\n", (p->partno + 1), filename, err ? "failed" : "ok");
             }
             else
             {
-                p->saveXML(p->loaded_file);
-                fprintf(stderr, "Part %d saved to %s\n", (p->partno + 1), p->loaded_file);
+                err = p->saveXML(p->loaded_file);
+                fprintf(stderr, "Part %d saved to %s: %s\n", (p->partno + 1), p->loaded_file, err ? "failed" : "ok");
             }
+            if (err)
+                d.reply("/alert", "s",
+                        "Failed To Write, please check file permissions");
```

Comment:
s/Failed To Write/Failed to save file/

???

Reply:
Yes, it makes sense, on the other hand, the function is mostly used when clicking OVERWRITE in READ mode in Fusion, so I chose 'Failed To Write' as a natural response to 'Overwrite'. But it would also be used when doing Save Instrument from the File menu (which however when testing right now doesn't seem to emit the /alert for whatever reason), which case 'Failed to save file' would be more appropriate, and considering the name of the OSC call (savexml) your suggestion also makes more sense.

I've got a couple of changes lined up for this PR based on comments from @fundamental so I'll add this change to that and send them all off together with any other changes that Mark might have regarding the functionality itself.